arctic-report-card/ARCRC.ttl

```diff
@@ -766,6 +936,152 @@ odo:ARCRC_00000068 rdf:type owl:Class ;
+                   obo:IAO_0000115 "A prominent socioeconomic development in recent years has been an increase in maritime ship traffic (characterized in view of ship movements and attributes of type, size, and flag state) in the Arctic Ocean as the sea ice diminishes with climate warming." ;
+                   obo:IAO_0000119 "ship traffic indicator as defined in the Arctic Report Card 2022 Essay at https://arctic.noaa.gov/report-card/report-card-2022/satellite-record-of-pan-arctic-maritime-ship-traffic/" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-11-21T19:43:23Z"^^xsd:dateTime ;
+                   rdfs:label "Satellite Record of Ship Traffic Indicator"@en .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000074
+odo:ARCRC_00000074 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000512 ;
+                   obo:IAO_0000115 "Lake ice cover is an important component of the cryosphere and changes to the timing of ice freeze-up, break-up, duration, and thickness can have substantial impacts on weather and climate, transportation, and ecology (e.g., Brown and Duguay 2010; Arp et al. 2019)." ;
+                   obo:IAO_0000119 "lake ice indicator as defined in the Arctic Report Card 2022 Essay at https://arctic.noaa.gov/report-card/report-card-2022/lake-ice/" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-11-21T19:43:23Z"^^xsd:dateTime ;
+                   rdfs:label "Lake Ice Indicator"@en .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000075
+odo:ARCRC_00000075 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000512 ;
+                   obo:IAO_0000115 "Arctic geese are keystone herbivores and their numbers and distributions are influenced by summer and winter environmental conditions and changes in forage quality and quantity." ;
+                   obo:IAO_0000119 "Arctic geese indicator as defined in the Arctic Report Card 2022 Essay at https://arctic.noaa.gov/report-card/report-card-2022/arctic-geese-of-north-america/" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-11-21T19:43:23Z"^^xsd:dateTime ;
+                   rdfs:label "Arctic Geese Indicator"@en .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000076
+odo:ARCRC_00000076 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000513 ;
+                   obo:IAO_0000115 "Arctic insects are sensitive indicators of ecological responses to climate change." ;
+                   obo:IAO_0000119 "Arctic Pollinators as defined in the Arctic Report Card 2022 Essay at https://arctic.noaa.gov/report-card/report-card-2022/arctic-pollinators/" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-07-30T19:43:23Z"^^xsd:dateTime ;
+                   rdfs:label "Arctic Pollinators"@en .
```

Comment:
Should it be "Arctic Pollinators Indicator" like the others?

Reply:
Since these were "Frostbite" essays, the other year's Frostbite essays didn't have indicators next to them, so I wrote it like this to match those. But I think changing them to have "Indicator" at the end would be good. I've changed it to Arctic Pollinators Indicator and Glacier & Permafrost Hazards Indicator. I left the previous year's "Western Alaska Salmon" and "Working with Communities to Observe the Arctic" unchanged though.